ci_framework/roles/edpm_build_images/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Install edpm-image-builder
+  ansible.builtin.include_tasks: install.yml
+
+- name: Find latest image name
+  import_role:
```

Comment:
```suggestion
  ansible.builtin.import_role:
```
Though this one is also called from the main playbook. Maybe we should check the availability of the facts, and import that role if and only if we're missing them?

Reply:
done